### ISSUE TITLE
✨ Set up Copilot instructions

### ISSUE THREAD

**Issue by sburdges-eng** [sburdges-eng/KmiDi#10]
use https://github.com/sburdges-eng/KmiDi/pull/6 as our basis for improvement unless builds have become a loss. use codespace currently open in repo to study refine and finish training. 75-200 epochs ( until near unrecoverable diminishing returns. train all models ensure the appropriate one complete generation of the spectocloud use '/Users/seanburdges/Desktop/Google Drive (Not synced)' '/Volumes/Extreme SSD' /Users/seanburdges/Desktop/README.md /Users/seanburdges/Desktop/KmiDi-remote /Users/seanburdges/Desktop/kelly-music-brain-backup /Users/seanburdges/Desktop/KmiDi '/Users/seanburdges/Desktop/New Folder With Items' '/Users/seanburdges/Desktop/final kel' '/Users/seanburdges/Desktop/KmiDi v.' /Users/seanburdges/Desktop/kelly-audio-data [sburdges@gmail.com](mailto:sburdges@gmail.com) googlr drive, one drive, icloud, for any training dataset you could benafit from. aqcuitre additional data sets from github public domains if needed reference pr #7 and 8 9
